AST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/3MFReader/ThreeMFWorkspaceReader.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/FirmwareUpdateChecker/FirmwareUpdateCheckerJob.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/FirmwareUpdateChecker/FirmwareUpdateCheckerLookup.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/FirmwareUpdateChecker/FirmwareUpdateChecker.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/PostProcessingPlugin/Script.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/USBPrinting/USBPrinterOutputDevice.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/SimulationView/SimulationView.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/Toolbox/src/Toolbox.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/Toolbox/src/PackagesModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/Toolbox/src/AuthorsModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/FirmwareUpdater/FirmwareUpdaterMachineAction.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/X3DReader/X3DReader.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/GCodeReader/FlavorParser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/VersionUpgrade/VersionUpgrade21to22/Profile.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/Models.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/UM3PrintJobOutputModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/DiscoverUM3Action.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/SendMaterialJob.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/ClusterUM3OutputDevice.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/CuraEngineBackend/CuraEngineBackend.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/plugins/CuraEngineBackend/StartSliceJob.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PreviewPass.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/CuraApplication.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrintInformation.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PickingPass.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutputDevice.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/BuildVolume.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/MachineActionManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/API/Account.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/TaskManagement/OnExitCallbackManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/NetworkedPrinterOutputDevice.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/PrintJobOutputModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/ConfigurationModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/NetworkMJPGImage.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/GenericOutputController.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/ExtruderOutputModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/PrinterOutputModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/PrinterOutput/ExtruderConfigurationModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/SettingVisibilityPreset.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/SettingInheritanceManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/GlobalStack.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/CuraContainerStack.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/ExtrudersModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/ContainerManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/MachineManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Settings/ExtruderManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Backups/Backup.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Scene/ConvexHullDecorator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Scene/GCodeListDecorator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Arranging/ArrangeObjectsAllBuildPlatesJob.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/OAuth2/AuthorizationService.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/OAuth2/AuthorizationRequestHandler.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/OAuth2/LocalAuthorizationServer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/ContainerNode.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/MaterialGroup.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/VariantManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/QualityGroup.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/MaterialNode.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/QualityManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/MaterialManager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/QualityNode.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/cura/Machines/Models/SettingVisibilityPresetsModel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/Cura/scripts/check_gcode_buffer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/securedrop/securedrop/crypto_util.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/pylint/pylint/test/functional/unused_typing_imports.pyInvalid bindings, s at ('body', 4, 'body', 1, 'body', 1) refers to Namespace(pos=('body', 4, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/pylint/pylint/test/functional/self_cls_assignment.pyInvalid bindings, c at ('body', 2, 'body', 2) refers to Namespace(pos=('body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/pylint/pylint/test/functional/used_before_assignment_nonlocal.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/django-extensions/tests/management/commands/shell_plus_tests/test_utils.pyInvalid bindings, n at ('body', 9, 'body', 11) refers to Namespace(pos=('body', 9), kind=<Kind.Function: 2>), but is not defined there, data/repos/graphene/tests/intense/api_stress.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/project.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/application.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/events.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/config.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/registry.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/highlighting.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/sphinx/sphinx/setup_command.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/osutil.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/parallel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/docutils.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/logging.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/util/stemmer/porter.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/_epub_base.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/text.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/linkcheck.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/texinfo.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/gettext.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/html.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/latex/transforms.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/builders/latex/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/search/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/directives/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/coverage.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/graphviz.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/doctest.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/autosummary/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/autodoc/directive.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/autodoc/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/autodoc/importer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/napoleon/iterators.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/ext/napoleon/docstring.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/pycode/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/pycode/parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/transforms/post_transforms/images.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/transforms/post_transforms/code.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/environment/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/domains/cpp.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/domains/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/writers/text.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/writers/texinfo.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sphinx/sphinx/writers/latex.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/cookiejar.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/tracing.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_request.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/client_proto.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/base_protocol.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_runner.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/streams.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/connector.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_protocol.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/test_utils.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/http_websocket.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_response.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_app.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_exceptions.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_fileresponse.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/client_ws.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/http_writer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_ws.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/multipart.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/http_parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/client_reqrep.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/helpers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_server.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/formdata.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/worker.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/payload.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_routedef.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/web_urldispatcher.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/aiohttp/aiohttp/locks.pyInvalid bindings, s at ('body', 8, 'body', 1) refers to Namespace(pos=('body', 8), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_web_protocol.pyInvalid bindings, c at ('body', 14, 'body', 1) refers to Namespace(pos=('body', 14), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_client_ws_functional.pyInvalid bindings, u at ('body', 97, 'body', 2) refers to Namespace(pos=('body', 97), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_client_functional.pyInvalid bindings, s at ('body', 11, 'body', 11, 'body', 9) refers to Namespace(pos=('body', 11, 'body', 11), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_proxy.pyInvalid bindings, n at ('body', 10, 'body', 2) refers to Namespace(pos=('body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_client_connection.pyInvalid bindings, s at ('body', 18, 'body', 1) refers to Namespace(pos=('body', 18), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_client_session.pyInvalid bindings, c at ('body', 20, 'body', 6) refers to Namespace(pos=('body', 20), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_proxy_functional.pyInvalid bindings, s at ('body', 16, 'body', 1) refers to Namespace(pos=('body', 16), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_web_websocket_functional.pyInvalid bindings, e at ('body', 37, 'body', 2) refers to Namespace(pos=('body', 37), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_web_functional.pyInvalid bindings, t at ('body', 43, 'body', 3) refers to Namespace(pos=('body', 43), kind=<Kind.Function: 2>), but is not defined there, data/repos/aiohttp/tests/test_web_sendfile_functional.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/action_space_walker.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/worlds/wikitables_world.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/worlds/quarel_world.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/worlds/wikitables_variable_free_world.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/worlds/nlvr_world.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/worlds/world.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/type_declarations/type_declaration.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/contexts/atis_sql_table_context.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/contexts/table_question_context.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/contexts/sql_context_utils.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/contexts/table_question_knowledge_graph.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/executors/nlvr_executor.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/semparse/executors/wikitables_sempre_executor.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/training/trainer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/training/metrics/bleu.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/training/metrics/attachment_scores.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/training/metrics/span_based_f1_measure.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/modules/elmo.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/modules/encoder_base.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/modules/seq2seq_encoders/stacked_self_attention.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/modules/seq2vec_encoders/cnn_highway_encoder.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/state_machines/states/checklist_statelet.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/models/semantic_parsing/nlvr/nlvr_coverage_semantic_parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/models/semantic_parsing/wikitables/wikitables_erm_semantic_parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/dataset.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/vocabulary.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/token_indexers/pos_tag_indexer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/token_indexers/dep_label_indexer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/token_indexers/openai_transformer_byte_pair_indexer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/dataset_readers/language_modeling.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/dataset_readers/simple_language_modeling.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/dataset_readers/semantic_parsing/quarel.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/fields/adjacency_field.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/fields/text_field.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/fields/knowledge_graph_field.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/fields/list_field.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/iterators/multiprocess_iterator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/allennlp/allennlp/data/iterators/data_iterator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/pipenv/pipenv/patched/notpip/_internal/configuration.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/pipenv/pipenv/vendor/tomlkit/container.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zilencer/management/commands/profile_request.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/middleware.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/decorator.pyInvalid bindings, c at ('body', 9, 'body', 3, 'body', 3) refers to Namespace(pos=('body', 9, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/zulip/zerver/tests/test_queue.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tests/test_queue_worker.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tests/test_events.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/zulip/zerver/tests/test_slack_importer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tests/test_report.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tests/test_tornado.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/data_import/hipchat_user.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/data_import/hipchat_subscriber.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/data_import/hipchat_attachment.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/data_import/sequencer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/worker/queue_processors.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/request.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/queue.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/export.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/unminify.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/outgoing_webhook.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/timeout.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/stream_recipient.pyInvalid bindings, c at ('body', 254, 'body', 1) refers to Namespace(pos=('body', 254), kind=<Kind.Function: 2>), but is not defined there, data/repos/zulip/zerver/lib/actions.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/test_helpers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/test_runner.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/db.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/openapi.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/exceptions.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/bugdown/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/lib/bugdown/fenced_code.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/views/messages.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/views/streams.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/ioloop_logging.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/handlers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/websocket_client.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/event_queue.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/socket.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/zerver/tornado/exceptions.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/analytics/tests/test_counts.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/run-dev.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/tests/test_linter_custom_check.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/lib/html_grep.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/lib/graph.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/lib/html_branches.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zulip/tools/documentation_crawler/documentation_crawler/spiders/check_help_documentation.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/concurrent_scanner.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/plugins/certificate_info_plugin.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/plugins/http_headers_plugin.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/plugins/utils/trust_store/trust_store.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/cli/json_output.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/cli/output_hub.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/utils/connection_helpers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/sslyze/sslyze/utils/thread_pool.pyInvalid bindings, u at ('body', 28, 'body', 5, 'body', 2, 'body', 1) refers to Namespace(pos=('body', 28, 'body', 5, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/dcos/release/__init__.pyInvalid bindings, u at ('body', 10, 'body', 2) refers to Namespace(pos=('body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/dcos/packages/dcos-history/extra/test_history_service.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/dpu-utils/dpu_utils/mlutils/vocabulary.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/dpu-utils/dpu_utils/utils/chunkwriter.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/dpu-utils/dpu_utils/tfutils/tfvariablesaver.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/dpu-utils/dpu_utils/tfutils/gradratiologgingoptimizer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/zmirror/tests/base_class.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/electrum/electrum/synchronizer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/electrum/electrum/address_synchronizer.pyInvalid bindings, e at ('body', 19, 'body', 6, 'body', 2) refers to Namespace(pos=('body', 19, 'body', 6), kind=<Kind.Function: 2>), but is not defined there, data/repos/electrum/electrum/base_wizard.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/electrum/electrum/daemon.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/electrum/electrum/websockets.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/electrum/electrum/blockchain.pyInvalid bindings, o at ('body', 17, 'body', 10, 'body', 13) refers to Namespace(pos=('body', 17, 'body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/electrum/electrum/plugins/trustedcoin/qt.pyInvalid bindings, o at ('body', 42, 'body', 148, 'body', 22) refers to Namespace(pos=('body', 42, 'body', 148), kind=<Kind.Function: 2>), but is not defined there, data/repos/electrum/electrum/gui/qt/main_window.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/electrum/electrum/gui/kivy/main_window.pyInvalid bindings, n at ('body', 11, 'body', 9, 'body', 1, 'body', 2) refers to Namespace(pos=('body', 11, 'body', 9), kind=<Kind.Function: 2>), but is not defined there, data/repos/electrum/electrum/gui/kivy/uix/dialogs/settings.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/doc/development/source/orange-demo/orangedemo/OWLearningCurveC.pyInconsistent parent namespace for _ in Namespace(pos=('body', 10, 'body', 1, 'body', 1), kind=<Kind.Function: 2>): Namespace(pos=('body', 10, 'body', 1, 'body', 1), kind=<Kind.Function: 2>) and Namespace(pos=('body', 10, 'body', 1), kind=<Kind.Function: 2>), data/repos/orange3/Orange/tests/test_evaluation_testing.pyInvalid bindings, v at ('body', 7, 'body', 1, 'body', 1) refers to Namespace(pos=('body', 7, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/preprocess/continuize.pyInvalid bindings, d at ('body', 74, 'body', 2) refers to Namespace(pos=('body', 74), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/gui.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/owsilhouetteplot.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/owpythagoreanforest.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_owlinearprojection.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_owpythagoreanforest.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_ownomogram.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_owsilhouetteplot.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_owheatmap.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/visualize/tests/test_owpythagorastree.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/owpredictions.pyInvalid bindings, c at ('body', 35, 'body', 2) refers to Namespace(pos=('body', 35), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/evaluate/owrocanalysis.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/tests/test_owrocanalysis.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/tests/test_owpredictions.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/tests/test_owcalibrationplot.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/tests/test_owliftcurve.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/evaluate/tests/test_owtestlearners.pyInvalid bindings, d at ('body', 10, 'body', 19, 'body', 1) refers to Namespace(pos=('body', 10, 'body', 19), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/unsupervised/owdistances.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/owlouvainclustering.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/owmds.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/owhierarchicalclustering.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/owkmeans.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/tests/test_owmanifoldlearning.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/tests/test_owkmeans.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/tests/test_owmds.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/unsupervised/tests/test_owpca.pyInvalid bindings, o at ('body', 11, 'body', 1) refers to Namespace(pos=('body', 11), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/utils/listfilter.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/utils/messages.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/utils/combobox.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/utils/messagewidget.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/owimpute.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/owselectcolumns.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/orange3/Orange/widgets/data/owrank.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/owfeaturestatistics.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/oweditdomain.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/owdatasets.pyInvalid bindings, d at ('body', 17, 'body', 28, 'body', 3) refers to Namespace(pos=('body', 17, 'body', 28), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/data/owcreateclass.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owpaintdata.pyInvalid bindings, c at ('body', 27, 'body', 28, 'body', 1, 'body', 0) refers to Namespace(pos=('body', 27, 'body', 28), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/widgets/data/tests/test_owfile.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owimpute.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owdatasampler.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owsave.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owselectrows.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/widgets/data/tests/test_owrank.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/canvas/scheme/widgetsscheme.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/orange3/Orange/canvas/application/addons.pyInvalid bindings, s at ('body', 39, 'body', 12, 'body', 5) refers to Namespace(pos=('body', 39, 'body', 12), kind=<Kind.Function: 2>), but is not defined there, data/repos/orange3/Orange/data/io.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/pip/src/pip/_internal/configuration.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/apps/blender/verification_task.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/apps/blender/blender_reference_generator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/apps/blender/verification_queue.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/apps/core/task/coretaskstate.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/apps/core/task/coretask.pyInconsistent parent namespace for _ in Namespace(pos=('body', 19, 'body', 2, 'body', 1), kind=<Kind.Function: 2>): Namespace(pos=('body', 19, 'body', 2, 'body', 1), kind=<Kind.Function: 2>) and Namespace(pos=('body', 19, 'body', 2), kind=<Kind.Function: 2>), data/repos/golem/tests/golem/resource/hyperdrive/test_hyperdrive_resourcemanager.pyInvalid bindings, h at ('body', 33, 'body', 5, 'body', 2) refers to Namespace(pos=('body', 33, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/golem/tests/golem/task/test_taskmanager.pyInvalid bindings, d at ('body', 29, 'body', 4, 'body', 8) refers to Namespace(pos=('body', 29, 'body', 4), kind=<Kind.Function: 2>), but is not defined there, data/repos/golem/scripts/concent_acceptance_tests/base.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/node.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/client.pyInvalid bindings, u at ('body', 2, 'body', 0) refers to Namespace(pos=('body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/golem/golem/rpc/utils.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/rpc/session.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/p2p/node.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/p2p/p2pservice.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/concent/filetransfers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/concent/handlers_library.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/concent/client.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/network/transport/tcpserver.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/docker/task_thread.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/docker/config.pyInvalid bindings, r at ('body', 16, 'body', 3, 'body', 2) refers to Namespace(pos=('body', 16, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/golem/golem/resource/client.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/ethereum/transactionsystem.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/taskstate.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/taskkeeper.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/tasksession.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/taskcomputer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/taskrequestorstats.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/localcomputer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/golem/golem/task/taskmanager.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/rasa_core/rasa_core/trackers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/rasa_core/rasa_core/training/dsl.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-recoveryservicesbackup/tests/recoveryservicesbackup_testcase.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_workspaces.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_experiments.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_quota_and_usage.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_file_servers.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_clusters.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/azure-sdk-for-python/azure-mgmt-batchai/tests/test_mgmt_batchai_jobs.pyInvalid bindings, a at ('body', 64, 'body', 10, 'body', 23) refers to Namespace(pos=('body', 64, 'body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Tools/clinic/clinic.pyInvalid bindings, l at ('body', 30, 'body', 4) refers to Namespace(pos=('body', 30), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/sre_compile.pyInvalid bindings, w at ('body', 5, 'body', 3, 'body', 2, 'body', 0) refers to Namespace(pos=('body', 5, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/unittest/test/test_assertions.pyInvalid bindings, s at ('body', 8, 'body', 9, 'body', 0, 'body', 0) refers to Namespace(pos=('body', 8, 'body', 9), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/unittest/test/test_result.pyInvalid bindings, w at ('body', 11, 'body', 42, 'body', 3) refers to Namespace(pos=('body', 11, 'body', 42), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_exceptions.pyInvalid bindings, f at ('body', 12, 'body', 38, 'body', 1, 'body', 2) refers to Namespace(pos=('body', 12, 'body', 38), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_dataclasses.pyInvalid bindings, s at ('body', 18, 'body', 1, 'body', 1) refers to Namespace(pos=('body', 18, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_thread.pyInvalid bindings, i at ('body', 36, 'body', 2, 'body', 2) refers to Namespace(pos=('body', 36, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test__xxsubinterpreters.pyInvalid bindings, e at ('body', 112, 'body', 5, 'body', 4) refers to Namespace(pos=('body', 112, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/_test_multiprocessing.pyInvalid bindings, f at ('body', 14, 'body', 0, 'body', 3) refers to Namespace(pos=('body', 14, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_tokenize.pyInvalid bindings, s at ('body', 22, 'body', 14, 'body', 3) refers to Namespace(pos=('body', 22, 'body', 14), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_urllib2_localnet.pyInvalid bindings, s at ('body', 9, 'body', 6, 'body', 2) refers to Namespace(pos=('body', 9, 'body', 6), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_contextlib_async.pyInvalid bindings, o at ('body', 81, 'body', 13, 'body', 8) refers to Namespace(pos=('body', 81, 'body', 13), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_io.pyInvalid bindings, s at ('body', 37, 'body', 2, 'body', 1, 'body', 0) refers to Namespace(pos=('body', 37, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_nntplib.pyInvalid bindings, _ at ('body', 45, 'body', 0, 'body', 29) refers to Namespace(pos=('body', 45, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_functools.pyInvalid bindings, p at ('body', 10, 'body', 3, 'body', 5) refers to Namespace(pos=('body', 10, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_threading_local.pyInvalid bindings, c at ('body', 74, 'body', 0, 'body', 1) refers to Namespace(pos=('body', 74, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_concurrent_futures.pyInvalid bindings, r at ('body', 15, 'body', 33, 'body', 1) refers to Namespace(pos=('body', 15, 'body', 33), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_tcl.pyInvalid bindings, r at ('body', 29, 'body', 32, 'body', 2, 'body', 0) refers to Namespace(pos=('body', 29, 'body', 32), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_httplib.pyInvalid bindings, f at ('body', 26, 'body', 6, 'body', 2) refers to Namespace(pos=('body', 26, 'body', 6), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_sax.pyInvalid bindings, C at ('body', 17, 'body', 19, 'body', 1) refers to Namespace(pos=('body', 17, 'body', 19), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_coroutines.pyInconsistent parent namespace for s in Namespace(pos=('body', 31, 'body', 1, 'body', 11), kind=<Kind.Function: 2>): Namespace(pos=('body', 31, 'body', 1, 'body', 11), kind=<Kind.Function: 2>) and Namespace(pos=('body', 31, 'body', 1), kind=<Kind.Function: 2>), data/repos/cpython/Lib/test/test_socketserver.pyInvalid bindings, f at ('body', 27, 'body', 25, 'body', 3) refers to Namespace(pos=('body', 27, 'body', 25), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_enum.pyInvalid bindings, e at ('body', 5, 'body', 5, 'body', 1) refers to Namespace(pos=('body', 5, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_frame.pyInvalid bindings, c at ('body', 5, 'body', 2, 'body', 1) refers to Namespace(pos=('body', 5, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_urllib_response.pyInvalid bindings, D at ('body', 11, 'body', 8, 'body', 5) refers to Namespace(pos=('body', 11, 'body', 8), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncgen.pyInvalid bindings, r at ('body', 28, 'body', 3, 'body', 0) refers to Namespace(pos=('body', 28, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_httpservers.pyInvalid bindings, f at ('body', 48, 'body', 2, 'body', 1) refers to Namespace(pos=('body', 48, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_xml_etree.pyInvalid bindings, r at ('body', 27, 'body', 10, 'body', 11) refers to Namespace(pos=('body', 27, 'body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_wsgiref.pyInvalid bindings, d at ('body', 10, 'body', 15, 'body', 1) refers to Namespace(pos=('body', 10, 'body', 15), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_contextlib.pyInvalid bindings, f at ('body', 11, 'body', 0, 'body', 0) refers to Namespace(pos=('body', 11, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_generators.pyInvalid bindings, r at ('body', 4, 'body', 31, 'body', 1) refers to Namespace(pos=('body', 4, 'body', 31), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_yield_from.pyInvalid bindings, b at ('body', 10, 'body', 7, 'body', 2) refers to Namespace(pos=('body', 10, 'body', 7), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_importlib/test_api.pyInvalid bindings, m at ('body', 33, 'body', 5, 'body', 3) refers to Namespace(pos=('body', 33, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_warnings/__init__.pyInconsistent parent namespace for e in Namespace(pos=('body', 25, 'body', 28, 'body', 1, 'body', 1), kind=<Kind.Function: 2>): Namespace(pos=('body', 25, 'body', 28, 'body', 1, 'body', 1), kind=<Kind.Function: 2>) and Namespace(pos=('body', 25, 'body', 28), kind=<Kind.Function: 2>), data/repos/cpython/Lib/test/test_import/__init__.pyInvalid bindings, d at ('body', 219, 'body', 0, 'body', 0) refers to Namespace(pos=('body', 219), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/support/__init__.pyInvalid bindings, l at ('body', 9, 'body', 9, 'body', 3) refers to Namespace(pos=('body', 9, 'body', 9), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_locks.pyInvalid bindings, k at ('body', 16, 'body', 16, 'body', 0, 'body', 3) refers to Namespace(pos=('body', 16, 'body', 16, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_subprocess.pyInconsistent parent namespace for _ in Namespace(pos=('body', 28, 'body', 100, 'body', 8), kind=<Kind.Function: 2>): Namespace(pos=('body', 28, 'body', 100, 'body', 8), kind=<Kind.Function: 2>) and Namespace(pos=('body', 28, 'body', 100), kind=<Kind.Function: 2>), data/repos/cpython/Lib/test/test_asyncio/test_tasks.pyInvalid bindings, c at ('body', 39, 'body', 10, 'body', 1) refers to Namespace(pos=('body', 39, 'body', 10), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_events.pyInvalid bindings, c at ('body', 16, 'body', 5, 'body', 5, 'body', 1) refers to Namespace(pos=('body', 16, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_sslproto.pyInvalid bindings, l at ('body', 6, 'body', 5, 'body', 2) refers to Namespace(pos=('body', 6, 'body', 5), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_runners.pyInvalid bindings, f at ('body', 8, 'body', 2, 'body', 5) refers to Namespace(pos=('body', 8, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_queues.pyInvalid bindings, d at ('body', 5, 'body', 1, 'body', 1, 'body', 1) refers to Namespace(pos=('body', 5, 'body', 1, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_buffered_proto.pyInvalid bindings, e at ('body', 22, 'body', 12, 'body', 5, 'body', 0) refers to Namespace(pos=('body', 22, 'body', 12), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/test/test_asyncio/test_futures.pyInvalid bindings, c at ('body', 31, 'body', 6) refers to Namespace(pos=('body', 31), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/asyncio/tasks.pyInvalid bindings, c at ('body', 5, 'body', 3, 'body', 1) refers to Namespace(pos=('body', 5, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/cpython/Lib/tkinter/test/test_tkinter/test_misc.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-postfix/certbot_postfix/installer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-postfix/certbot_postfix/postconf.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-apache/certbot_apache/configurator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-apache/certbot_apache/http_01.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-apache/certbot_apache/parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-apache/certbot_apache/tests/http_01_test.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/certbot/certbot/crypto_util.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/error_handler.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/cli.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/plugins/selection_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/plugins/standalone_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/plugins/webroot.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/plugins/common.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/plugins/manual.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/tests/auth_handler_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/tests/hook_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot/tests/display/completer_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/acme/acme/crypto_util_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/acme/acme/standalone_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/acme/acme/standalone.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/acme/acme/client.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-dns-route53/certbot_dns_route53/dns_route53.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-nginx/certbot_nginx/parser_obj.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-nginx/certbot_nginx/configurator.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/certbot/certbot-nginx/certbot_nginx/parser.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/RxPY/rx/streams/singlestream.pyInvalid bindings, r at ('body', 2, 'body', 4, 'body', 1) refers to Namespace(pos=('body', 2, 'body', 4), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/take.pyInvalid bindings, d at ('body', 2, 'body', 1) refers to Namespace(pos=('body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/takelastwithtime.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/RxPY/rx/operators/observable/filter.pyInvalid bindings, i at ('body', 4, 'body', 2, 'body', 4) refers to Namespace(pos=('body', 4, 'body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/concat.pyInvalid bindings, s at ('body', 4, 'body', 1, 'body', 4) refers to Namespace(pos=('body', 4, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/switchlatest.pyInvalid bindings, r at ('body', 3, 'body', 2) refers to Namespace(pos=('body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/range.pyInvalid bindings, d at ('body', 4, 'body', 2) refers to Namespace(pos=('body', 4), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/skiplastwithtime.pyInvalid bindings, l at ('body', 3, 'body', 1, 'body', 2) refers to Namespace(pos=('body', 3, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/timeinterval.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/RxPY/rx/operators/observable/map.pyInvalid bindings, s at ('body', 2, 'body', 2) refers to Namespace(pos=('body', 2), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/scan.pyInvalid bindings, s at ('body', 1, 'body', 1) refers to Namespace(pos=('body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/toiterable.pyInvalid bindings, r at ('body', 2, 'body', 3, 'body', 1) refers to Namespace(pos=('body', 2, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/skip.pyInvalid bindings, d at ('body', 5, 'body', 1, 'body', 3) refers to Namespace(pos=('body', 5, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/fromiterable.pyInvalid bindings, o at ('body', 5, 'body', 1, 'body', 1) refers to Namespace(pos=('body', 5, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/returnvalue.pyInvalid bindings, i at ('body', 6, 'body', 1, 'body', 2) refers to Namespace(pos=('body', 6, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/zip.pyInvalid bindings, d at ('body', 6, 'body', 0) refers to Namespace(pos=('body', 6), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/timer.pyInvalid bindings, d at ('body', 7, 'body', 0) refers to Namespace(pos=('body', 7), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/rx/operators/observable/delay.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/RxPY/rx/subjects/subject.pyInvalid bindings, r at ('body', 6, 'body', 1, 'body', 1, 'body', 2) refers to Namespace(pos=('body', 6, 'body', 1, 'body', 1), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/tests/test_concurrency/test_mainloopscheduler/py3_asyncioscheduler.pyInvalid bindings, r at ('body', 16, 'body', 0, 'body', 2) refers to Namespace(pos=('body', 16, 'body', 0), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/tests/test_observable/py3_tofuture.pyInvalid bindings, c at ('body', 10, 'body', 3, 'body', 3) refers to Namespace(pos=('body', 10, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/tests/test_observable/test_doaction.pyInvalid bindings, i at ('body', 10, 'body', 0, 'body', 3, 'body', 0) refers to Namespace(pos=('body', 10, 'body', 0, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/RxPY/tests/test_observable/test_takewhile.pyInvalid bindings, u at ('body', 16, 'body', 2) refers to Namespace(pos=('body', 16), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/tests/helpers/test_entity.pyInvalid bindings, m at ('body', 9, 'body', 23, 'body', 5) refers to Namespace(pos=('body', 9, 'body', 23), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/tests/components/test_influxdb.pyInvalid bindings, i at ('body', 4, 'body', 2) refers to Namespace(pos=('body', 4), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/tests/components/http/__init__.pyInvalid bindings, c at ('body', 65, 'body', 5) refers to Namespace(pos=('body', 65), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/tests/components/device_tracker/test_owntracks.pyInvalid bindings, c at ('body', 12, 'body', 2) refers to Namespace(pos=('body', 12), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/tests/scripts/test_auth.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/data_entry_flow.pyInvalid bindings, h at ('body', 54, 'body', 2) refers to Namespace(pos=('body', 54), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/core.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/requirements.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/config_entries.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/util/aiohttp.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/home-assistant/homeassistant/util/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/util/logging.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/helpers/intent.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/helpers/entity_values.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/helpers/restore_state.pyInvalid bindings, c at ('body', 20, 'body', 3) refers to Namespace(pos=('body', 20), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/helpers/condition.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/helpers/state.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/auth/auth_store.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/auth/providers/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/auth/providers/homeassistant.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/auth/mfa_modules/notify.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/auth/mfa_modules/totp.pyInvalid bindings, r at ('body', 46, 'body', 13) refers to Namespace(pos=('body', 46), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/alarmdecoder.pyInvalid bindings, p at ('body', 18, 'body', 5) refers to Namespace(pos=('body', 18), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/edp_redy.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/mqtt/__init__.pyInvalid bindings, r at ('body', 22, 'body', 4, 'body', 6) refers to Namespace(pos=('body', 22, 'body', 4), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/cloud/iot.pyInvalid bindings, s at ('body', 42, 'body', 17) refers to Namespace(pos=('body', 42), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/emulated_hue/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/sensor/fints.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/light/switch.pyInvalid bindings, p at ('body', 22, 'body', 10) refers to Namespace(pos=('body', 22), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/light/hue.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/light/group.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/media_player/cast.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/recorder/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/home-assistant/homeassistant/components/device_tracker/bbox.pyInvalid bindings, h at ('body', 15, 'body', 9) refers to Namespace(pos=('body', 15), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/climate/zhong_hong.pyInvalid bindings, p at ('body', 15, 'body', 6) refers to Namespace(pos=('body', 15), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/components/switch/unifi.pyInvalid bindings, c at ('body', 13, 'body', 4) refers to Namespace(pos=('body', 13), kind=<Kind.Function: 2>), but is not defined there, data/repos/home-assistant/homeassistant/scripts/benchmark/__init__.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/analyzer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/local.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/package.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/deploy/deployer.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/deploy/planner.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/deploy/executor.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/chalice/chalice/cli/filewatch/stat.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/errbot/errbot/plugin_manager.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/connexion/tests/test_references.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/connexion/tests/test_operation2.pyInvalid bindings, e at ('body', 6, 'body', 3, 'body', 1) refers to Namespace(pos=('body', 6, 'body', 3), kind=<Kind.Function: 2>), but is not defined there, data/repos/hypothesis-python/tests/py3/test_asyncio.pyInvalid bindings, D at ('body', 17, 'body', 8, 'body', 5) refers to Namespace(pos=('body', 17, 'body', 8), kind=<Kind.Function: 2>), but is not defined there, data/repos/cython/tests/run/test_asyncgen.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/websocket.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/tcpserver.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/httputil.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/testing.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/simple_httpclient.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/process.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/template.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/ioloop.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/tcpclient.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/http1connection.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/httpserver.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/httpclient.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/curl_httpclient.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/log.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/gen.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/queues.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/options.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/iostream.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/util.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/locks.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/routing.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/simple_httpclient_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/locale_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/locks_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/websocket_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/iostream_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/tcpclient_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/web_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/gen_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/httpserver_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/test/routing_test.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/platform/caresresolver.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/tornado/tornado/platform/asyncio.pyInvalid bindings, c at ('body', 14, 'body', 2) refers to Namespace(pos=('body', 14), kind=<Kind.Function: 2>), but is not defined there, data/repos/mypy/misc/analyze_cache.pyInvalid bindings, c at ('body', 15, 'body', 1) refers to Namespace(pos=('body', 15), kind=<Kind.Function: 2>), but is not defined there, data/repos/mypy/misc/touch_checker.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/misc/test_case_to_actual.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/test-data/samples/crawl2.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/test-data/samples/crawl.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/test-data/stdlib-samples/3.2/tempfile.pyAST node of type <class 'typed_ast._ast3.Tuple'> has no name, data/repos/mypy/test-data/stdlib-samples/3.2/test/test_set.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/test-data/stdlib-samples/3.2/test/test_pprint.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/test-data/stdlib-samples/3.2/test/test_shutil.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/modulefinder.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/semanal_pass3.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/build.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/tvar_scope.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/report.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/stats.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/gclogger.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/binder.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/checkexpr.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/find_sources.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/nodes.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/typeanal.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/mypyc_hacks.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/strconv.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/treetransform.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/indirection.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/dmypy_server.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/stubgen.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/semanal.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/fswatcher.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/fastparse.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/options.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/types.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/type_visitor.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/util.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/fastparse2.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/scope.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/fscache.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/test/testmerge.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/test/visitors.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/test/testpep561.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/test/data.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/server/subexpr.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/server/update.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/server/aststrip.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/mypy/mypy/server/deps.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/rasa_nlu/rasa_nlu/model.pyAST node of type <class 'typed_ast._ast3.Attribute'> has no name, data/repos/rasa_nlu/rasa_nlu/emulators/__init__.py